code/modules/surgery/organs/organ_external.dm

```diff
@@ -164,6 +164,27 @@
 	if(!HAS_TRAIT(owner, TRAIT_IB_IMMUNE))
 		limb_flags &= ~CANNOT_INT_BLEED
 
+/obj/item/organ/external/attack(mob/M, mob/living/user)
```

Comment:
How does this interact with initiating surgery? Does it override it?

Reply:
testing

Reply:
doesnt overide